internal/model/model.go

```diff
@@ -0,0 +1,68 @@
+package model
+
+import (
+	"net"
+
+	"github.com/google/uuid"
+	"github.com/metal-toolbox/bioscfg/internal/log"
+)
+
+type (
+	StoreKind string
+)
+
+const (
+	AppName    = "bioscfg"
+	AppSubject = "biosControl"
+
+	InventoryStoreYAML StoreKind = "yaml"
```

Comment:
why do we need different `StoreKind`s?

Reply:
That was copy and pasted from another service. I removed the whole `StoreKind` since we really don't have different store kinds yet.